client/signup/steps/theme-selection/index.jsx

```diff
@@ -76,7 +76,7 @@ class ThemeSelectionStep extends Component {
 		const { translate } = this.props;
 		const headerText = translate( 'Choose a theme.' );
 		const subHeaderText = translate(
-			'No need to overthink it. You can always switch to a different theme later.',
+			'Pick one of our popular themes to get started or choose from hundreds more after you sign up.',
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?